workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -64,12 +64,20 @@ public void Awake()
                 var workerId =
                     CommandLineUtility.GetCommandLineValue(commandLineArgs, RuntimeConfigNames.WorkerId,
                         string.Empty);
+                var useDynamicWorkerId = CommandLineUtility.GetCommandLineValue(commandLineArgs,
```

Comment:
It's kind of odd to be dealing with `string` rather than `bool`.

Reply:
I agree, will take a look within CommandLineUtility